setup/README.adoc

```diff
@@ -0,0 +1,85 @@
+= Dev Sandbox Setup
+
+This document describes how to setup a Dev Sandbox environment that is similar to production and can be used as part of an onboarding process for new operators.
+
+=== Prereqs
+
+1. Provision an OCP cluster with sufficient resources.
+
+For example, if using the openshift-install tool to install OCP on AWS you can use an install-config.yaml file that has the following configuration:
+
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+  name: worker
+  platform: {}
+  replicas: 3
+controlPlane:
+  hyperthreading: Enabled
+  name: master
+  platform:
+    aws:
+      type: "m5.4xlarge"
+  replicas: 3
+compute:
+- hyperthreading: Enabled
+  name: worker
+  platform:
+    aws:
+      type: "m5.2xlarge"
+  replicas: 3
+metadata:
+  name: sandbox-test
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  aws:
+    region: us-east-1
+publish: External
+----
+
+2. Install your operator(s)
+
+3. Create a user-workloads.yaml file. This file should contain resources that the setup tool will automatically create in user namespaces. Use the sample user-workloads.yaml 🚧 coming soon 🚧  as a base and add custom resources that your operator uses.
+
+4. Complete the following steps:
+* Make sure you have set the `QUAY_NAMESPACE` variable: `export QUAY_NAMESPACE=<quay-username>`
+* Log in to the quay.io using `docker login quay.io`
+* Make sure that the visibility of all repositories `host-operator`, `member-operator` and `registration-service` in quay is set to `public` (https://quay.io/repository/<your-username>/host-operator?tab=settings https://quay.io/repository/<your-username>/member-operator?tab=settings https://quay.io/repository/<your-username>/registration-service?tab=settings)
+* Log in to the target OpenShift 4.2+ cluster with cluster admin privileges using `oc login`
+
+=== Dev Sandbox Setup
+
+🚧 This section is still a work in progress 🚧
+
+1. Clone this repository
+`git clone git@github.com:codeready-toolchain/toolchain-e2e.git`
+2. Run the following to install the Dev Sandbox operators
+`make dev-deploy-e2e`
+3. // TODO instructions for provisioning a large amount of users to simulate load on the cluster
+
+=== Evaluate the Cluster and Operator(s)
+
+With the cluster now under load, it's time to evaluate the environment.
+
+1. Use your operators as a user would and evaluate the performance.
+2. Monitor the cluster's performance using the Monitoring view in the OpenShift Console.
+
+=== Clean up
+
+==== Remove Only Users and Their Namespaces
+
+Run `oc get usersignup -n toolchain-host-operator -o jsonpath='{.items..metadata.name}' | xargs oc delete usersignup`
```

Comment:
couldn't you use:
```suggestion
Run `oc get usersignup -n toolchain-host-operator -o name | xargs oc delete usersignup`
```

Reply:
or just:
```suggestion
Run `oc delete usersignups --all --all-namespaces`
```

you could also add a makefile target for that - it should be pretty easy - there is already one that contains the necessary commands: https://github.com/codeready-toolchain/toolchain-e2e/blob/master/make/clean.mk#L13-L14

Reply:
Thanks, adding a target for that is a great suggestion! 🥇  Added `clean-users` target in 3ee5c179b731f5c786807e4e1a5a919d78d863ed